pprocutils/errors.go

```diff
@@ -30,6 +30,8 @@ const (
 	ErrorCodeSubjectNotFound
 	ErrorCodeVersionNotFound
 	ErrorCodeInvalidSchema
+
+	ErrorCodeInternal
```

Comment:
I would move this to line 29 after `ErrorCodeMemoryOutOfRange`, it belongs more in that group. The first 4 error codes were already in the last release so we can't change those codes, but we can safely change the schema error codes, since those were not released yet.